evalml/tests/component_tests/test_per_column_imputer.py

```diff
@@ -262,7 +262,7 @@ def test_per_column_imputer_woodwork_custom_overrides_returned_by_components(
     override_types = [Integer, Double, Categorical, NaturalLanguage, Boolean]
     for logical_type in override_types:
         # Column with Nans to boolean used to fail. Now it doesn't
-        if has_nan and logical_type == Boolean:
+        if has_nan and logical_type in [Boolean, NaturalLanguage]:
```

Comment:
Leave out NaturalLanguage since casting this will result in `np.nan` becoming `pd.NA`, which fails the imputer

Reply:
But how come this wouldn't happen before? Woodwork doesn't convert `np.nan` to `pd.NA` in 0.4.2? Is this because of the pandas upgrade?

Reply:
@freddyaboulton I believe it's due to the upgrade!
![image](https://user-images.githubusercontent.com/22552445/125699351-e5e0b7c4-6365-423f-937e-3430b87e80c9.png)
Looking at the 1.3.0 release docs, seems like there's a [lot of changes](https://pandas.pydata.org/pandas-docs/stable/user_guide/missing_data.html) with NaN handling, and they're using [`<NA>` for scalar types](https://pandas.pydata.org/pandas-docs/stable/user_guide/missing_data.html#experimental-na-scalar-to-denote-missing-values)

Reply:
Thanks for looking into this @bchen1116 Let's list this as a breaking change for now. I imagine we might want to file an issue to discuss if there are any changes we need to make to the simpleimputer? If users run it on natural language after this pr they'll get a stacktrace they didn't get before.

Reply:
@freddyaboulton updated the release notes with the breaking change and filed the issue [here](https://github.com/alteryx/evalml/issues/2515)!